src/DayPicker/DayPicker.js

```diff
@@ -211,7 +211,7 @@ module.exports = kind(
 		if (str) {
 			return str;
 		}
-
+		this.selectedIndex.sort();
```

Comment:
I realize that this line exists in `ExpandablePicker` as well, but I'm curious if we know how the ordering of the selected indices becomes incorrect? It would be nice to avoid sorting here, in general, if we can correct the core logic.